include/cinder/Log.h

```diff
@@ -35,6 +35,11 @@
 #include <memory>
 #include <mutex>
 
+#if defined( CINDER_MSW )
+#include <Windows.h>
```

Comment:
Can these windows-specific headers be pushed to the .cpp? [CinderMswFwd.h](https://github.com/cinder/Cinder/blob/glNext/include/cinder/msw/CinderWindowsFwd.h) might help, but if not hopefully we can forward declare whatever we need so that Windows.h doesn't get pulled into user app files.